368 further condensation

mhagdorn opened the pull request:
I have condensed the paper a little more, especially with reducing the number of references in mind. With PR #375 this should reduce the number of references to 16. We need to remove another 4 references. Maybe merge #375 first and then see if we can identify another 4 candidates for removal. Lengthwise we should be getting there.

Each change (removal) is a separate commit to allow cherry picking.

jpthiele commented on summarised_competencies.md:
```diff
@@ -243,7 +231,7 @@ We highlight that professional conduct requires these creations to be reliable a
 Independent of the encoding into the respective national law in an RSE's jurisdiction,
 the right to information privacy is internationally recognised as a fundamental human right,
-e.g., in the European Convention on Human Rights [@CouncilOfEurope-ETS005-2021; @Hirvela2022].
+e.g., in the European Convention on Human Rights.
```
sneaky :)

MakisH replied:
The citation is completely removed in the current state.

jpthiele commented:
Small correction on the numbers. 
We have to remove 5 more references if we want to be able to cite the full version.

mhagdorn commented:
> Small correction on the numbers. We have to remove 5 more references if we want to be able to cite the full version.

I figured we would drop the ref to the preprint on arxiv

CaptainSifff commented:
I've just merged #375

CaptainSifff commented:
this version is in my workflow at 7298 words. the limit is 6250....

CaptainSifff commented:
the ref will most likely be replaced by the f1000 ref.

mhagdorn commented:
I had another go at shaving off some words and paragraphs. I also dropped some more refs. I converted the pdf to a text file using pdftotext and removed the table of contents. We are now at 6133 words and 12 refs. Again, I made a separate commit for each chunk for easier review. 12 references are not many.

jpthiele commented:
Awesome, thank you!

MakisH commented on summarised_competencies.md:
```diff
@@ -574,9 +544,9 @@ Open science practices are increasingly required by research funding agencies.
-and delegating tasks to project members (e.g. \gls{USERS}, and \gls{TEAM}).
+Large research software projects require somebody to
+ maintain the overall perspective (\gls{PM})
+and to delegate tasks to project members (e.g. \gls{USERS}, and \gls{TEAM}).
```
There are still a few skill codes here, which I understand should be removed (#381).

mhagdorn replied:
I didn't take out the skill codes because I think they are useful to tie the skills and specialisations together. The thing that was removed is the glossary which expanded the skill codes. Hovering over the codes is broken though. So we do have to think what we are going to do.

MakisH replied:
Opened a separate issue as a reminder when converting to the other template: #383

MakisH reviewed (approved):
I took the liberty to directly edit some parts, to save some time. Please check my commits, should be rather small and targetted.

I mainly added more hints that this is a summary (now also in the abstract and conclusion), and I also explained why we decided for this (at the end of the introduction). I also modified a bit the biography and conclusion to hint towards the broader work.

I read throught the whole thing and it is self-consistent. If anything, I still think that the Values and Current Challenges sections are a bit disconnected to the rest of the text, as well as that the paragraphs themselves could be better connected between them.

Generally good to go as-is, see next the #383.

Thanks a lot for the extensive edits and effort to condense! :pray:

mhagdorn commented:
@MakisH, thanks for going through it again and all the changes. I spotted some bits which I changed. I think we are good unless we've cut something that others think should be in there.

One thing I did notice is that we haven't mentioned the Dagstuhl thing. Maybe that is something we want to add? This could go into the 3rd para of the introduction along with all the workshops @jpthiele?

CaptainSifff commented:
looks good, I would merge this. Or do we need to discuss sth. more here?

jpthiele commented:
LGTM :+1:

MakisH commented:
@mhagdorn all good updates, thanks! :+1: 

> One thing I did notice is that we haven't mentioned the Dagstuhl thing. Maybe that is something we want to add? This could go into the 3rd para of the introduction along with all the workshops @jpthiele?

I added this to the post-processing issue: https://github.com/the-teachingRSE-project/competencies/issues/383

> looks good, I would merge this. Or do we need to discuss sth. more here?

LGTM as well :+1:
